packages/block-editor/src/components/block-preview/README.md

```diff
@@ -29,3 +29,17 @@ A block instance (object) or a blocks array you would like to render a preview.
 Width of the preview container in pixels. Controls at what size the blocks will be rendered inside the preview.
 
 `viewportWidth` can be used to simulate how blocks look on different device sizes or to make sure make sure multiple previews will be rendered with the same scale, regardless of their content.
+
+### `onReady`
```

Comment:
The preview component suffered many changes recently. Would it be worth to make the prop experimental for two versions? Until we stabilize a little bit on its API.

Reply:
Agree. let me update it to `__experimentalOnReady_.